samples/web/content/apprtc/js/appcontroller.js

```diff
@@ -266,9 +271,11 @@ AppController.prototype.transitionToWaiting_ = function() {
 
   this.transitionToWaitingTimer_ = setTimeout(function() {
```

Comment:
One way to solve this would be to call this remoteVideoResetTimer, and always cancel it when setting remoteVideo to a stream.

I think we can get rid of the code block at line 243 with this change.